job.js

```diff
@@ -147,8 +178,12 @@ class BigQueryJob {
             const [job] = await this.bigQuery.createQueryJob(jobOpts);
             this.logger.info('bigquery-job.js Executed job. Getting query results. name:%s', this.name);
 
-            const [rows] = await this._getQueryResults(job);
-            this.logger.info('bigquery-job.js Got query results. name:%s totalRows:%s elapsed:%s ms', this.name, rows.length, elapsed.end());
+            let rows;
+
+            if(this.avoidReturningResults !== true){
+                [rows] = await this._getQueryResults(job);
+                this.logger.info('bigquery-job.js Got query results. name:%s totalRows:%s elapsed:%s ms', this.name, rows.length, elapsed.end());
+            }
```

Comment:
Add an else with a log informing that we are avoiding the query results and inform destination table with write disposition